Gopkg.toml

```diff
@@ -37,30 +37,30 @@ required = [
 
 [[constraint]]
   # Also defined in bazelbuild/rules_go
-  # https://github.com/bazelbuild/rules_go/blob/436452edc29a2f1e0edc22d180fbb57c27e6d0af/go/private/repositories.bzl#L75
-  version = "1.1.0"
+  # https://github.com/bazelbuild/rules_go/blob/109c520465fcb418f2c4be967f3744d959ad66d3/go/private/repositories.bzl#L52
+  revision = "aa810b61a9c79d51363740d207bb46cf8e620ed5"
```

Comment:
`master: Could not introduce github.com/grpc-ecosystem/grpc-gateway@master, as it has a dependency on github.com/golang/protobuf with constraint aa810b61a9c79d51363740d207bb46cf8e620ed5, which has no overlap with existing constraint master from github.com/grpc-ecosystem/go-grpc-middleware@master`

broken!

Reply:
I'm not a `go dep` but I don't think this change introduced the problem, as even before the version was constrained and would have conflicted with `go-grpc-middleware`'s constraint. @javasgl Maybe you need to use an override to resolve the difference?

Reply:
Yeah I've explained the situation in #829